ABC/contest.go

```diff
@@ -0,0 +1,48 @@
+package ABC
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+type Post struct {
+	ID               string `json:"id"`
+	StartEpochSecond int    `json:"start_epoch_second"`
+	DurationSecond   int64  `json:"duration_second"`
+	Title            string `json:"title"`
+	RateChange       string `json:"rate_change"`
+}
+
+func max(a int, b int) int {
+	if a >= b {
+		return a
+	} else {
+		return b
+	}
+}
+
+func Id() int {
+	var posts []Post
+
+	resp, _ := http.Get("https://kenkoooo.com/atcoder/resources/contests.json")
+
+	body, _ := io.ReadAll(resp.Body)
+
+	json.Unmarshal(body, &posts)
+	res := 1
+	for idx := range posts {
```

Comment:
```suggestion
	for i, post := range posts {
```

index アクセスする必要があまりなさそうなので、こうした方が良さそう